oscpy/parser.py

```diff
@@ -96,6 +111,7 @@ def parse_blob(value, offset=0, **kwargs):
     (int, (b'i', b'i')),
     (bytes, (b's', b'%is')),
     (bytearray, (b'b', b'%ib')),
+    (tuple, (b'm', b'I'))
```

Comment:
matching a tuple might be a little overreaching there, looking at the other extended types, i guess some could be represented by tuples, like rgba color. Maybe it would be better to have a class or namedtuple type to describe these more complex data and do less guessing, i actually pondered about how to solve that when adding the blob writer, bytearray seemed like a decent fit, but it was still some guessing about best suitability, other extended osc types would certainly benefit from a way to hint the type (as well as pass the output to osc routes handlers.

Reply:
You are correct. Also a more explicit typing-approach could be beneficial since numerous OSC types overlap with built-in python types (i.e. python int objects can represent both 64-bit and 32-bit OSC integers). I will address this comment with a commit.